spe10/SPE10.md

```diff
@@ -1,19 +1,88 @@
-SPE10-MOD02-02| Two-Phase, with Cartesian regular grid.                       | MODEL-02   | Yes     | Results are comparable to other vendors as presented in the paper. E100 not tested. 
-
-**Notes:** 
-
-1.   _Results Match_ column indicate if the OPM Flow results match the published results, see the SPE10.odp document for comparisons.
-2.   Model one cases run with ten day time steps for comparison purposes.
-3.   Model two used variable time steps and requires CPR to run, see the parameter files.
-
-**Version: 12 August 2022**
-                                                                                                                                  
\ No newline at end of file
+Case Name     | Case Desciption                                  | Base Model | Test<br />Type | Results<br />Match | Comments |
+---------     | -----------------------------                    | ---------- | ---- | ------- | ------------------------------------- |
+SPE10-MOD01-01| Three-Phase, with Cartesian regular grid.        | MODEL-01   |      | Yes     | Results are comparable to other vendors as presented in the paper. however, E100 fails to run to completion.
+SPE10-MOD01-02| Two-Phase, with Cartesian regular grid.          | MODEL-01   | Int  | Yes     | Results are comparable to other vendors as presented in the paper; however, E100 fails to run to completion.
+SPE10-MOD01-03| Three-Phase, with Corner-Point regular grid.     | MODEL-01   |      | Yes     | Results are comparable to other vendors as presented in the paper. E100 not tested.
+SPE10-MOD01-04| Two-Phase, with Corner-Point regular grid.       | MODEL-01   |      | Yes     | Results are comparable to other vendors as presented in the paper. E100 not tested.
+SPE10-MOD02-01| Three-Phase, with Cartesian regular grid.        | MODEL-02   |      | Yes     | Results are comparable to other vendors as presented in the paper. E100 not tested.
+SPE10-MOD02-02| Two-Phase, with Cartesian regular grid.          | MODEL-02   | Int  | Yes     | Results are comparable to other vendors as presented in the paper. E100 not tested.
+
+Both models are based on the SPE 10 comparative SPE paper:
+
+   "Tenth SPE comparative solution project: a comparison of upscaling techniques"
+   By Mike Christie (Heriot-Wat University) and Martin Blunt (Imperial College)
+   SPE, August 2001.
+
+Originally, these tests were derived when OPM Flow was only able to run three-phase models, and thus these tests were
+formulated as pseudo two-phase cases. For 2022, the tests have been re-factored with corrections, and OPM Flow results
+have been compared with the published results. Both two-phase and three-phase OPM Flow cases give exactly the same
+results, and the results are comparable to the published results from various vendors.
+
+**Notes:**
+
+1. _Test Type_ column shows if the the case is used for intergration testing (_Int_), or regression teststing (_Reg_)
```

Comment:
Tiny nit: "... if THE THE case is ..."

Reply:
@bska I don't know for sure, so I'll defer to someone wiser than me. Note that none of the plots are taken from the actual papers, the data was digitized and the plots in the documentation were derived from the digitized data. Since we state the source, then I don't think it is an issue. But like I said, I'll defer to someone wiser.

Will fix the typo.